### ISSUE TITLE
Array inputs

### ISSUE THREAD

**Issue by novhack** [setzer22/blackjack#55]
Some nodes like "Merge meshes" would really benefit from having just one input array (in this case an array of meshes). Blender geometry nodes have this feature and indicate it with a "pill" slot.
![image](https://user-images.githubusercontent.com/42614907/197421053-6c14116c-86ef-4827-8d01-9933f00e26ec.png)

**Comment by setzer22** [comment]
This is planned, but is currently blocked on this PR of egui_node_graph https://github.com/setzer22/egui_node_graph/pull/30 which is currently being reworked by @mxgrey.

**Comment by Zireael07** [comment]
I see this PR also blocks/makes more difficult the idea of a loop node (that would allow executing several nodes X times)

**Comment by mxgrey** [comment]
I'm doing a rework to achieve multi-input/output that's using a totally separate approach from the PR that was linked. See the [last comment](https://github.com/setzer22/egui_node_graph/pull/30#issuecomment-1247931215) in the PR thread.

I'll be making sure the version that I'm working on is compatible with looped graphs, because that's an important requirement for my use cases.

**Comment by setzer22** [comment]
Yes, I should've clarified :sweat_smile:, the code for original linked PR went stale, but the ongoing discussion in the thread is still relevant :+1:
